docs/integrations/ai-engines/lightwood.mdx

```diff
@@ -142,6 +142,54 @@ USING
                 ]};
 ```
 
+## Predictions' Explanation
```

Comment:
Nit: maybe rename section to "Explainability"

Reply:
"Predictions' Explainability" or just "Explainability"?

Reply:
Just "Explainability", but not feeling too strong either way